tests/integration/features/istio/production/main-suite/istio_observability.feature

```diff
@@ -0,0 +1,50 @@
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "connection_termination_details"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "upstream_transport_failure_reason"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "bytes_received"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "bytes_sent"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "duration"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "upstream_service_time"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "x_forwarded_for"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "user_agent"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "request_id"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "authority"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "upstream_host"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "upstream_cluster"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "upstream_local_address"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "downstream_local_address"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "downstream_remote_address"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "requested_server_name"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "route_name"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "traceparent"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "tracestate"
+
+  Scenario: Istio interacts with the otel collector service defined in meshconfig
+    Given Logging and tracing is enabled for the mesh using providers "stdout-json" for logs and "kyma-traces" for traces
+    And Httpbin application "otel-collector-mock" is running in namespace "kyma-system"
```

Comment:
Do you think it would be helpful to have a comment like:
```suggestion
  # For a simpler setup we use a httpbin application as OpenTelemetry collector, because we only want to verify that the OpenTelemetry API is called by checking the access logs of the httpbin.
    And Httpbin application "otel-collector-mock" is running in namespace "kyma-system"
```

Reply:
Definitely, thanks